R/utils-ci.R

```diff
@@ -60,8 +62,7 @@ compose_start <- function(connect_license = Sys.getenv("RSC_LICENSE"), rsc_versi
   compose_path <- find_compose()
 
   license_details <- determine_license_env(connect_license)
-  compose_file <- switch(
-    license_details$type,
+  compose_file <- switch(license_details$type,
     "file" = "ci/test-connect-lic.yml",
```

Comment:
Future: We should squash these compose files into one (or remove them entirely) and simply provide the correct bind mounts and environment variables to the container as it starts.